Dockerfile-rhel8

```diff
@@ -0,0 +1,94 @@
+FROM registry.access.redhat.com/ubi8
```

Comment:
There are some registiries listed here different than what is included in this pr.
https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image

I used `registry.access.redhat.com/ubi8/ubi` but I think they are the same as what you used.

I also tried to use ubi-minimal but it failed with the following.
```
sed: can't read /etc/dnf/plugins/subscription-manager.conf: No such file or directory
The command '/bin/sh -c sed -i 's/^enabled=.*/enabled=0/g' /etc/dnf/plugins/subscription-manager.conf' returned a non-zero code: 2
```